src/idl/src/parser.y

```diff
@@ -814,8 +814,10 @@ case_label:
   ;
 
 element_spec:
-    type_spec declarator
-      { TRY(idl_create_case(pstate, LOC(@1.first, @2.last), $1, $2, &$$)); }
+    annotations type_spec declarator
+      { TRY(idl_create_case(pstate, LOC(@1.first, @3.last), $2, $3, &$$));
+        TRY_EXCEPT(idl_annotate(pstate, $$, $1), free($$));
+      }
```

Comment:
Can union element-spec non-terminals be annotated? I was under the impression that only the switch-type-spec and the union itself can be annotated? Perhaps due to the `@id` and `@hashid` annotations?

Reply:
Maybe you have a link to an example or the spec or something?

Reply:
This is described in the XTypes v1.3 spec table 21.

I will also add this to the files.

Reply:
Much appreciated. The spec is rather unclear where annotations can/cannot be applied so :+1: